### PR TITLE
Add bento64 subdomain

### PR DESCRIPTION
**Pull request by nhandt64**
### Requirements
- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. *This requirement is optional if you are registering a domain for emails.*
- [x] The website is reachable.
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.
   - *You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.*

### Link to Website

https://bento64.github.io

### CODE REVIEW

**Review by wdhdev — APPROVED** [review]

**Comment by bento64** [comment]
Sorry, I have misconfigured Github Pages so it kept redirect to unassigned subdomain `bento64.is-a.dev`. I have reverted the configuration so the original Github Page at https://bento64.github.io should now be correctly displayed.
Please take a look again.
Thanks a lot!

**Comment by WilliamDavidHarrison** [comment]
It's all good. Just wait for a maintainer to merge it now

**Comment by WilliamDavidHarrison** [comment]
It's all good. Just wait for a maintainer to merge it now

**Pull request merged**
